### PR TITLE
Update to Cadence v0.27.0

### PR DESCRIPTION
**Pull request by turbolent**
______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to GitHub issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-emulator/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the GitHub PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: server/backend/backend.go [hunk 1]
```diff
@@ -217,14 +217,14 @@ func (b *Backend) SendTransaction(ctx context.Context, tx sdk.Transaction) error
 		case *types.FlowError:
 			// TODO - confirm these
 			switch t.FlowError.(type) {
-			case *fvmerrors.AccountAuthorizationError,
```
Comment (sideninja) [inline server/backend/backend.go:220]: does the dereference happen implicitly here?

Comment (turbolent) [reply server/backend/backend.go:220]: No, it seems like the types have been changed from pointer to non-pointer error types.

File: server/server.go [hunk 2]
```diff
@@ -141,7 +142,7 @@ func NewEmulatorServer(logger *logrus.Logger, conf *Config) *EmulatorServer {
 		"FlowServiceAccount": chain.ServiceAddress().HexWithPrefix(),
 		"FlowToken":          fvm.FlowTokenAddress(chain).HexWithPrefix(),
 		"FungibleToken":      fvm.FungibleTokenAddress(chain).HexWithPrefix(),
-		"FlowFees":           fvm.FlowFeesAddress(chain).HexWithPrefix(),
+		"FlowFees":           environment.FlowFeesAddress(chain).HexWithPrefix(),
```
Comment (bluesign) [inline server/server.go:145]: is there a logic for what is on fvm and what is on environment?

**Review by sideninja — APPROVED** [review]

File: server/server.go (hunk 2 above)
Comment (turbolent) [reply server/server.go:145]: I don't know why it was moved and just adjusted the code here to compile, but will ask the team.

Comment (janezpodhostnik) [reply server/server.go:145]: The FlowFeesAddress was moved together with its only usage in fvm/environment during a refactor.

There was [preparation done](https://github.com/onflow/flow-go/pull/3218) to remove all of these actually.  There will be a different way to get them soon: `systemcontracts.SystemContractsForChain(chain.ChainID())` which should clean this up.

**Pull request merged**